src/preact.d.ts

```diff
@@ -5,7 +5,9 @@ declare namespace preact {
 	}
 
 	interface PreactHTMLAttributes {
+		dangerouslySetInnerHTML?:Object;
```

Comment:
I can't help you there! haha
If it's just a naming thing, doesn't matter to me.